doc/api/util.md

```diff
@@ -894,6 +894,534 @@ encoded bytes.
 
 The encoding supported by the `TextEncoder` instance. Always set to `'utf-8'`.
 
+## util.types
```

Comment:
Just noting that this will have a named-import-in-ESM issue just like the `fs.promises` addition.